transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventArray.java

```diff
@@ -89,52 +94,77 @@ private void checkSize() {
     void realloc(boolean throwIfFail) {
         // Double the capacity while it is "sufficiently small", and otherwise increase by 50%.
         int newLength = capacity <= 65536 ? capacity << 1 : capacity + capacity >> 1;
-        long newMemoryAddress = PlatformDependent.reallocateMemory(memoryAddress, newLength * KQUEUE_EVENT_SIZE);
```

Comment:
Just to confirm, we didn't need to copy the old contents of the memory like was done previously?

Reply:
Good point... let me copy.

Reply:
fixed